web/themes/custom/ys_admin_theme/ys_admin_theme.info.yml

```diff
@@ -0,0 +1,37 @@
+name: YaleSites Admin Theme
+type: theme
+base theme: gin
+description: "Admin theme for YaleSites."
+core_version_requirement: ^8.9 || ^9 || ^10
+
+libraries:
+  - claro/global-styling
+  - gin/gin
+  - ys_admin_theme/paragraphs
+
+# Regions
+regions:
+  header: "Header"
+  pre_content: "Pre-content"
+  breadcrumb: Breadcrumb
+  highlighted: Highlighted
+  help: Help
+  content: Content
+  page_top: "Page top"
+  page_bottom: "Page bottom"
+  sidebar_first: "First sidebar"
+regions_hidden:
+  - sidebar_first
+
+components:
+  namespaces:
+    tokens:
+      - /themes/custom/atomic/node_modules/@yalesites-org/component-library-twig/components/00-tokens
```

Comment:
I think this may be related to the lack of dev-ops stuff mentioned above. Getting this all working locally is A CHORE at the moment, so if you're testing on your local, there's probably a lot of information you're missing